EJ/Submissions/HW_1.py

```diff
@@ -0,0 +1,15 @@
+#Favorite Food
+FavoriteFood="Chicken"
+print(f"{FavoriteFood}")
+FavoriteFood="ChickenWings"
+print(f"{FavoriteFood}")
+#Int vs. Floats
+print(float(3.2591+2.12093))
+print(float(2.12093+3.2591))
+#Family Members Age Calculator
+Mom=37
```

Comment:
Hmm are you really going to make your moms age public like that?

Reply:
I don't think she cares all that much